src/main/java/com/synopsys/integration/alert/provider/blackduck/factories/BlackDuckValidatorFactory.java

```diff
@@ -0,0 +1,47 @@
+/**
+ * blackduck-alert
+ *
+ * Copyright (c) 2020 Synopsys, Inc.
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package com.synopsys.integration.alert.provider.blackduck.factories;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import com.synopsys.integration.alert.common.AlertProperties;
+import com.synopsys.integration.alert.common.persistence.accessor.SystemMessageUtility;
+import com.synopsys.integration.alert.provider.blackduck.BlackDuckProperties;
+import com.synopsys.integration.alert.provider.blackduck.BlackDuckValidator;
+
+@Component
+public class BlackDuckValidatorFactory {
+    private final AlertProperties alertProperties;
+    private final SystemMessageUtility systemMessageUtility;
+
+    @Autowired
+    public BlackDuckValidatorFactory(AlertProperties alertProperties, SystemMessageUtility systemMessageUtility) {
+        this.alertProperties = alertProperties;
+        this.systemMessageUtility = systemMessageUtility;
+    }
+
+    public BlackDuckValidator createValidator(BlackDuckProperties properties) {
```

Comment:
Can we put a TODO here to revisit this approach once we have everything else working?

Reply:
no longer needed.